environment.yml

```diff
@@ -17,4 +17,4 @@ dependencies:
     - deepdish
     - SimPEG
     - empymod
-    - git+https://github.com/geoscixyz/geosci-labs
+    - git+https://github.com/geoscixyz/geosci-labs.git
```

Comment:
the `.git` isn't needed here. please see the original and then this is good to go!